src/components/Categories/Categories.js

```diff
@@ -1,11 +1,21 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
 import Navigation from '../Navigation/Navigation';
+import { categAction } from '../../Redux/Categories/Categories';
 
-const Categories = () => (
-  <div>
-    <Navigation />
-    <button type="button">Check status</button>
-  </div>
-);
+const Categories = () => {
+  const dispatch = useDispatch();
+
+  const handleStatus = () => {
+    dispatch(categAction());
```

Comment:
- [ ] I kindly suggest that you check you categAction because  "Check status" button on categories page does not work as expected i.e it doesn't display "Under Construction" when it is clicked.